src/daemon.py

```diff
@@ -81,12 +84,49 @@ def __search_task(self, job_id: JobId) -> None:
         self.db.update_job_files(job_id, file_count)
         self.db.agent_start_job(self.group_id, job_id, iterator)
 
+    def __load_plugins(self) -> None:
+        active_plugins = []
+        for plugin_class in METADATA_PLUGINS:
+            plugin_name = plugin_class.get_name()
+            plugin_config = self.db.get_plugin_configuration(plugin_name)
+            try:
+                active_plugins.append(plugin_class(self.db, plugin_config))
+                logging.info(f"Loaded {plugin_name} plugin")
```

Comment:
nitpick but can't resist
```suggestion
                logging.info("Loaded %s plugin", plugin_name)
```